### ISSUE TITLE
Question: Can hubut react to emoji reactions with this adapter?

### ISSUE THREAD

**Issue by stpaultim** [zulip/hubot-zulip#14]
Can hubut react to emoji reactions with this adapter?

![Zulip_-_Backdrop_CMS](https://user-images.githubusercontent.com/3144571/72643502-6e4f6100-3934-11ea-95c6-7ad9068af641.png)

I'd like to use a specific emoji reaction to generate karma points in our channel. 

I haven't done much research yet on my own and will post an answer if I find one before someone else chimes in.

**Comment by timabbott** [comment]
I don't think this hubot adapter supports that, but it should be easy to add using the zulip API.
